lib/OpenGuides/Config.pm

```diff
@@ -151,7 +151,14 @@ sub _init {
             warn "Don't know what to do with variable '$var'";
         }
     }
-
+    # Override config from file or defaults if an environment variable
+    # is set for a variable. e.g OPENGUIDES_CONFIG_read_only
+    foreach my $var ( @variables ) {
+       my $envvar = "OPENGUIDES_CONFIG_" . $var;
```

Comment:
I personally think it would look better if the environment variable was all upper case.

Reply:
```suggestion
       my $envvar = "OPENGUIDES_CONFIG_" . uc ($var);
```
so this ?